torchreid/models/tresnet.py

```diff
@@ -0,0 +1,50 @@
+from .common import ModelInterface
+import timm
+
+__ALL__ = ['tresnet']
+
+class TResnetTimm(ModelInterface):
+    def __init__(self,
+                pretrained=False,
+                dropout_cls = None,
+                lr_finder=None,
+                num_classes=1000,
+                **kwargs):
+        super().__init__(**kwargs)
+        dropout_rate = dropout_cls.p
```

Comment:
This is due to timm repo interface. In the case of Resnet we can use our Dropout implementation, but in case of TResnet we cannot as it merged into the head

Reply:
And I decided to use our Dropout whenever it is possible.